### DISCUSSION TITLE
Question about react-native target?

### DISCUSSION THREAD

**Discussion by Morritz** [microsoft/react-native-windows#7922] [Q&A] [answered]
Is it currently possible to use just WPF, without any UWP stuff, and compile to binary that is compatible for ex. with windows 7 and every other version? If yes, what are the steps to reproduce such behaviour, is the template sufficient for that?

**Comment by chrisglein** [comment, marked as answer]
It is not possible to target have your React Native for Windows (RNW) app target WPF. It will use UWP. Some more information about that [here](https://microsoft.github.io/react-native-windows/docs/native-code#win32-desktop-apps-vs-rnw-apps). However it is possible to host an island of RNW in a WPF app. There's detail on that [here](https://techcommunity.microsoft.com/t5/windows-dev-appconsult/getting-started-with-react-native-for-windows/ba-p/912093). However note that RNW does not support Windows 7 (As per [this](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) it's Windows 10+).
